### ISSUE TITLE
App / fav Icon

### ISSUE THREAD

**Issue by frederikbugglin** [basxsoftwareassociation/basxconnect#98]

**Comment by saemideluxe** [comment]
Question: Is this issue about redesigning the basx Connect logo or to add a an option for users to customized the favicon? I think customization of the favicon is not necessary.

**Comment by frederikbugglin** [comment]
> Is this issue about redesigning the basx Connect logo

Yes, It is about the basx Connect Application logo as well as the favicon in the browser (and yes: I do see no need for the user to customize the favicon)
